lib/self-service/self-service-manager.js

```diff
@@ -49,7 +49,7 @@ const MGMT_USERS_PATH = "/cloud_directory/Users/";
 const MGMT_SIGN_UP_RESULT_PATH = "/cloud_directory/sign_up/confirmation_result";
 const MGMT_FORGOT_PASSWORD_RESULT_PATH = "/cloud_directory/forgot_password/confirmation_result";
 const MGMT_CHANGE_PASSWORD_PATH = "/cloud_directory/change_password";
-const PRODUCTION_IAM_TOKEN_URL = "https://iam.bluemix.net/oidc/token";
+const PRODUCTION_IAM_TOKEN_URL = "https://iam.test.cloud.ibm.com/identity/token";
```

Comment:
Shouldn't this be "https://iam.cloud.ibm.com/identity/token" ?

Reply:
fixed